src/components/App.js

```diff
@@ -139,18 +138,17 @@ class App extends Component<Props, State> {
   onEscape = (_, e) => {
     const {
       activeSearch,
+      quickOpenEnabled,
       closeActiveSearch,
-      closeQuickOpen,
-      quickOpenEnabled
+      closeQuickOpen
     } = this.props;
 
     if (activeSearch) {
       e.preventDefault();
       closeActiveSearch();
     }
 
-    if (quickOpenEnabled) {
-      e.preventDefault();
+    if (quickOpenEnabled === true) {
```

Comment:
Is there any reason for this change?

Reply:
Looks like this reverted this PR: https://github.com/devtools-html/debugger.html/pull/5866/files